src/browser-frontend/views/browser/chrome/tabbar/tab.jsx

```diff
@@ -73,6 +79,9 @@ export default class Tab extends PureComponent {
             }),
           } : {}),
         }}
+        draggable="true"
+        onDragStart={this.drag}
+        data={dragging}
```

Comment:
This is the first case of using data attributes instead of simple classes. Is there any particular reason for doing this? If you insist on using data attributes, React allows you to have `data-foo-bar` props on native DOM components which result in "foo-bar" attributes on the DOM element.